contributing-device-support.md

```diff
@@ -250,14 +250,27 @@ See [this](https://github.com/balena-io/contracts/pull/296) as an example to bas
 
 For private device types, balena will make the necessary changes when supplied with the hardware contract.
 
-## Step 4: Maintaining the repository and OS updates
+## Step 4: Setting up Hardware in the Loop Testing Pipeline
+
+In order to have the balenaOS image deployed to balenaCloud, it must first pass the [balenaOS automated test suite](./tests).
+This involves the following steps:
+
+1. Order [parts to build](https://github.com/balena-io-hardware/autokit-assembly-doc/blob/master/source/bom/bom.csv) the autokit test harness, [set it up](https://github.com/balena-io-hardware/autokit-assembly-doc/blob/master/README.md), and connect it to the board for which the CBS process is being done for. With the autokit setup, you would be able to run the automated test suite and test your balenaOS image on the board.
```

Comment:
@rcooke-warwick I assume you meant CDS instead of CBS here? And about that, should we even mention CDS?